src/repository/screens/repository.screen.js

```diff
@@ -95,12 +104,14 @@ class Repository extends Component {
   handlePress = index => {
     const {
       starred,
+      subscribed,
```

Comment:
I've found this `handlePress` method too fragile - depend on the index is not a good thing at all. We should improve this code.